test/e2e/case/native-protocols-kafka/satellite_config_kafka.yaml

```diff
@@ -0,0 +1,597 @@
+      flush_time: ${SATELLITE_LOGPIPE_SENDER_FLUSH_TIME:1000}
+      # The maximum buffer elements.
+      max_buffer_size: ${SATELLITE_LOGPIPE_SENDER_MAX_BUFFER_SIZE:200}
+      # The minimum flush elements.
+      min_flush_events: ${SATELLITE_LOGPIPE_SENDER_MIN_FLUSH_EVENTS:1}
+#      client_name: grpc-client
+#      forwarders:
+#        - plugin_name: native-log-grpc-forwarder
+      client_name: kafka-client
+      forwarders:
+        - plugin_name: native-log-kafka-forwarder
+          topic: ${ATELLITE_FORWARDER_KAFKA_TOPIC_LOGGING:skywalking-logs}
+  - common_config:
+      pipe_name: managementpipe
+    gatherer:
+      server_name: "grpc-server"
+      receiver:
+        plugin_name: "grpc-native-management-receiver"
+      queue:
+        plugin_name: "memory-queue"
+        # The maximum buffer event size.
+        event_buffer_size: ${SATELLITE_QUEUE_EVENT_BUFFER_SIZE:5000}
+        # The partition count of queue.
+        partition: ${SATELLITE_QUEUE_PARTITION:4}
+    processor:
+      filters:
+    sender:
+      fallbacker:
+        plugin_name: none-fallbacker
+      # The time interval between two flush operations. And the time unit is millisecond.
+      flush_time: ${SATELLITE_LOGMANAGEMENT_SENDER_FLUSH_TIME:1000}
+      # The maximum buffer elements.
+      max_buffer_size: ${SATELLITE_LOGMANAGEMENT_SENDER_MAX_BUFFER_SIZE:20}
+      # The minimum flush elements.
+      min_flush_events: ${SATELLITE_LOGMANAGEMENT_SENDER_MIN_FLUSH_EVENTS:1}
+#      client_name: grpc-client
+#      forwarders:
+#        - plugin_name: native-management-grpc-forwarder
+      client_name: kafka-client
+      forwarders:
+        - plugin_name: native-management-kafka-forwarder
+          topic: ${SATELLITE_FORWARDER_KAFKA_TOPIC_MANAGEMENT:skywalking-managements}
+  - common_config:
+      pipe_name: tracingpipe
+    gatherer:
+      server_name: "grpc-server"
+      receiver:
+        plugin_name: "grpc-native-tracing-receiver"
+      queue:
+        plugin_name: "memory-queue"
+        # The maximum buffer event size.
+        event_buffer_size: ${SATELLITE_QUEUE_EVENT_BUFFER_SIZE:5000}
+        # The partition count of queue.
+        partition: ${SATELLITE_QUEUE_PARTITION:4}
+    processor:
+      filters:
+    sender:
+      fallbacker:
+        plugin_name: none-fallbacker
+      # The time interval between two flush operations. And the time unit is millisecond.
+      flush_time: ${SATELLITE_TRACINGPIPE_SENDER_FLUSH_TIME:1000}
+      # The maximum buffer elements.
+      max_buffer_size: ${SATELLITE_TRACINGPIPE_SENDER_MAX_BUFFER_SIZE:200}
+      # The minimum flush elements.
+      min_flush_events: ${SATELLITE_TRACINGPIPE_SENDER_MIN_FLUSH_EVENTS:1}
+      client_name: kafka-client
+      forwarders:
+        - plugin_name: native-tracing-kafka-forwarder
+          topic: ${SATELLITE_FORWARDER_KAFKA_TOPIC_SEGMENT:skywalking-segments}
+  - common_config:
```

Comment:
There still have many not using pipelines, please remove them, to prevent abuse due to unknown circumstances. Such as profile/CDS/event/CLR/ALS pipe, etc.